mod/reportedcontent/views/default/reportedcontent/css.php

```diff
@@ -4,3 +4,6 @@
 .elgg-icon-report-this:before {
 	content: "\f071";
 }
+.elgg-menu-extras .elgg-menu-item-report-this .elgg-anchor-label {
+	display: none;
```

Comment:
I guess the mindset you need to have here is that this is a responsive stylesheet. A theme may decide to display the same menu items differently on a mobile device. Having only icons in the markup makes that a daunting task - you have to iterate through all the menus, figure out what the label should be etc.